master/buildbot/test/unit/test_scripts_upgrade_master.py

```diff
@@ -223,4 +223,6 @@ def test_upgradeDatabaseFail(self):
             mkconfig(basedir='test', quiet=True),
             config_module.MasterConfig())
         self.assertEqual(ret, 1)
-        self.assertIn("problem while upgrading!: o noz", sys.stderr.getvalue())
+        self.assertIn("problem while upgrading!:\n Traceback (most recent call last):\n",
```

Comment:
I wonder why there is space between `\n Traceback`? I think this test passes, so there should be space, but why?